src/routes/ml.ts

```diff
@@ -0,0 +1,55 @@
+import {
+    ML_MODELS,
+  } from '../config';
+  import { Request, Response, Router } from 'express';
+  import { readFileSync, renameSync, writeFileSync } from 'fs';
+
+  const router = Router();
+  
+  /**
+   * To swap model,
+   * 1. Get current model hash: GET /ml/hash?type=PVC
+   * 2. Upload new model to /tmp/pvc.onnx using POST /upload
+   * 3. Commit new model: POST /ml/commit with body { type: 'PVC', hash: 'new hash', path: '/tmp/pvc.onnx' }
+   */
+  router.get('/hash', async (req: Request, res: Response) => {
+    try {
+      if (!req.query.type) {
+        res.json({ error: 'model type is required' });
+        return;
+      }
+      // @ts-ignore
+      const modelPath = ML_MODELS[req.query.type];
+      if (!modelPath) {
+        res.json({ error: 'model type is not supported' });
+        return;
+      }
+      const hash = readFileSync(modelPath + '.hash', { encoding: 'utf-8' });
+      res.json({ hash: hash.trim() } );
```

Comment:
can we get an empty string if there's nothing we get back?
```suggestion
      const hash = readFileSync(modelPath + '.hash', { encoding: 'utf-8' });
      res.json({ hash: hash.trim() } );
```